src/app/containers/Article/index.jsx

```diff
@@ -11,46 +11,70 @@ import ArticleMain from '../ArticleMain';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import ConsentBanner from '../ConsentBanner';
+import { GhostWrapper, GridItemConstrainedMedium } from '../../lib/styledGrid';
 
 const logger = nodeLogger(__filename);
 
-/*
-  [1] This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
-*/
+const Container = ({ children, service }) => (
+  <Fragment>
+    <GlobalStyle />
+    <ServiceContextProvider service={service}>
+      <Helmet>
+        <link rel="manifest" href={`/${service}/articles/manifest.json`} />
+      </Helmet>
+      <ConsentBanner />
+      <HeaderContainer />
+      {children}
+    </ServiceContextProvider>
+  </Fragment>
+);
+
+Container.propTypes = {
+  children: node.isRequired,
+  service: string.isRequired,
+};
+
 const ArticleContainer = ({ loading, error, data, bbcOrigin }) => {
-  if (loading) return 'Loading...'; /* [1] */
+  if (loading) {
+    return (
+      <Container service="news">
```

Comment:
I dont this will work like this, as Container has ConsentBanner, without a Request context, and consent banner needs it https://github.com/bbc/simorgh/blob/latest/src/app/containers/ConsentBanner/index.jsx#L7